tests.py

```diff
@@ -14,6 +14,9 @@
 
 
 class TestVSM(unittest.TestCase):
+    def setUp(self):
+        print(self._testMethodName)
```

Comment:
You just need to run ./tests.py -v to see that, it's a feature of the unittest module, let's not duplicate it =)